File: app/src/main/java/com/app/rupiksha/adapters/PayoutBankAdapter.java
package com.app.rupiksha.adapters;

import android.app.Activity;
import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ArrayAdapter;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.app.rupiksha.R;
import com.app.rupiksha.models.PayoutAccountModel;

import java.util.List;

public class PayoutBankAdapter extends ArrayAdapter<PayoutAccountModel>
{
    private LayoutInflater layoutInflater;
    private Context mContext;
    private int gravity;

    public PayoutBankAdapter(@NonNull Context context, int resource, int textViewResourceId, @NonNull List<PayoutAccountModel> objects, int center)
    {
        super(context, resource, textViewResourceId, objects);
        layoutInflater = ((Activity) context).getLayoutInflater();
        mContext = context;
        this.gravity = center;
    }

    @NonNull
    @Override
    public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {

        if (convertView == null) {
            convertView = layoutInflater.inflate(R.layout.row_price_selector_item, parent, false);
        }
        PayoutAccountModel spinnerItem = getItem(position);
        TextView itemName = convertView.findViewById(R.id.tvItemName);
        itemName.setGravity(gravity);
        itemName.setText(spinnerItem.getAccount());
        return convertView;
    }

    @Override
    public View getDropDownView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {
        if (convertView == null) {
            convertView = layoutInflater.inflate(R.layout.row_price_selector_item_dropdown, parent, false);
        }
        PayoutAccountModel spinnerItem = getItem(position);
        TextView itemName = convertView.findViewById(R.id.tvItemName);
        itemName.setText(spinnerItem.getAccount());

        return convertView;
    }
}

File: app/src/main/java/com/app/rupiksha/adapters/PayoutBankAdapter.kt
package com.app.rupiksha.adapters

import android.app.Activity
import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ArrayAdapter
import android.widget.TextView
import com.app.rupiksha.R
import com.app.rupiksha.models.PayoutAccountModel

class PayoutBankAdapter(
    context: Context,
    resource: Int,
    textViewResourceId: Int,
    objects: List<PayoutAccountModel>,
    private val gravity: Int
) : ArrayAdapter<PayoutAccountModel>(context, resource, textViewResourceId, objects) {

    private val layoutInflater: LayoutInflater = (context as Activity).layoutInflater

    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
        val view = convertView ?: layoutInflater.inflate(R.layout.row_price_selector_item, parent, false)
        val spinnerItem = getItem(position)
        val itemName = view.findViewById<TextView>(R.id.tvItemName)
        itemName.gravity = gravity
        itemName.text = spinnerItem?.account
        return view
    }

    override fun getDropDownView(position: Int, convertView: View?, parent: ViewGroup): View {
        val view = convertView ?: layoutInflater.inflate(R.layout.row_price_selector_item_dropdown, parent, false)
        val spinnerItem = getItem(position)
        val itemName = view.findViewById<TextView>(R.id.tvItemName)
        itemName.text = spinnerItem?.account
        return view
    }
}
